dev/ci/cases/yamls/gefs_defaults_ci.yaml

```
@@ -7,5 +7,6 @@ base:
   USE_ATM_ENS_PERTURB_FILES: "NO"
   DO_BUFRSND: "NO"
   DO_GEMPAK: "NO"
+  DO_AWIPS: "YES"
```

I think this can be set to `NO` by default. 

```suggestion
  DO_AWIPS: "NO"
```